functions/network.js

```diff
@@ -2,16 +2,25 @@ import AxiosStrategy from "./strategies/AxiosStrategy";
 import FirefoxStrategy from "./strategies/FirefoxStrategy";
 import ChromeStrategy, { hasChromeExtensionInstalled } from "./strategies/ChromeStrategy";
 
+const isExtensionsAllowed = ({ state }) => {
+  console.log(typeof(state.postwoman.settings.EXTENSIONS_ENABLED) === 'undefined'
```

Comment:
Please comment out / remove `console.log()`

Reply:
Oops, forgot about that 😛

Reply:
Okay, fixed that in the latest commit.